packages/stockflux-watchlist/public/preview-chart.html

```diff
@@ -21,20 +24,22 @@
         display: flex;
         width: 100%;
         height: 100%;
-        box-sizing: border-box;
-        border: dotted 3px #fff;
-        overflow: hidden;
+        flex-direction: column;
         justify-content: center;
         align-items: center;
+        box-shadow: 0px 2px 4px 1px rgba(13, 36, 55, 1);
```

Comment:
Is this correct to the spec? The shadow values there are given as blur = 9 and spread = 3, so should it be
`box-shadow: 0px 2px 9px 3px` ?

Also, the spec lists the colour as #1B3B56, which I think is 27, 59, 86 in rgb?

Reply:
Changed the drop shadow values.